mkdocs.yml

```diff
@@ -26,16 +26,17 @@ theme:
 
 nav:
   - Home: 'index.md'
-  - Quickstart: 
-    - Quickstart: 'quickstart/quickstart.md'
-    - Initialization: 'quickstart/init.md'
-    - Build and Setup: 'quickstart/build_setup.md'
-    - Run: 'quickstart/server.md'
-    - Configuration: 'quickstart/config.md'
-    - Customization: 'quickstart/customize.md'
-    - Troubleshooting: 'quickstart/troubleshoot.md'
-    - Cleanup: 'quickstart/cleanup.md'
-  - Deployment:
+  - Install: 'install/index.md'
+  - Preview: 'preview/index.md'
+  - Develop:
```

Comment:
comment: I suspect this and deploy do not show on the side menu because they are not direct but with sub menus. See that install and preview are a one doc thing while this aren't. To be investigated later on.

Reply:
Also what I suspect, but there should be a way to make it show up...